script.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-mvn clean install -DskipTests
+mvn clean install -DskipTests -Dcheckstyle.skip=true
```

Comment:
Make sure to remove these flags before the PR gets merged

Reply:
Sure. I just wanted to test the cachesize unit test in travis